### PR TITLE
updating domain fronted roundtrip for receiving op name

### PR DESCRIPTION
**Pull request by WendelHime**

### CODE REVIEW

File: proxied/fronted.go [hunk 1]
```diff
@@ -14,15 +15,16 @@ const DefaultMasqueradeTimeout = 5 * time.Minute
 // fronting.
 //
 // Leave masqueradeTimeout as 0 to use a default value.
-func Fronted(masqueradeTimeout time.Duration) http.RoundTripper {
+func Fronted(opName string, masqueradeTimeout time.Duration) http.RoundTripper {
```
Comment (hwh33) [inline proxied/fronted.go:18]: I like this!

File: issue/issue.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ var (
 	maxLogSize = 10247680
 
 	client = &http.Client{
-		Transport: proxied.Fronted(0),
+		Transport: proxied.Fronted("issue_fronted_roundtrip", 0),
```
Comment (hwh33) [inline issue/issue.go:25]: Oh, good call 👍

**Review by hwh33 — APPROVED** [review]
LGTM!  (I realize this is still pending the [sampling discussion](https://github.com/getlantern/engineering/issues/1723#issuecomment-2451975960)).

File: chained/broflake_impl.go [hunk 3]
```diff
@@ -140,7 +140,8 @@ func makeBroflakeOptions(pc *config.ProxyConfig) (
 	// Broflake's HTTP client isn't currently configurable via PluggableTransportSettings, and so
 	// we just give it this domain fronted client in all cases
 	wo.HttpClient = &http.Client{
-		Transport: proxied.Fronted(masqueradeTimeout),
+		Transport: proxied.Fronted("broflake_fronted_roundtrip", masqueradeTimeout),
+		Timeout:   60 * time.Second,
```
Comment (WendelHime) [inline chained/broflake_impl.go:144]: @hwh33 is 60s good enough?

Comment (hwh33) [reply chained/broflake_impl.go:144]: Yep!

File: embeddedconfig/global.yaml.tmpl [hunk 4]
```diff
@@ -422,3 +422,4 @@ otel:
     autoupdate_download: 1
     autoupdate_install: 1
     check_update: 1
+    broflake_fronted_roundtrip: 800000
```
Comment (hwh33) [inline embeddedconfig/global.yaml.tmpl:425]: Linking to the discussion about this for posterity:
https://github.com/getlantern/engineering/issues/1723#issuecomment-2451975960

**Review by hwh33 — APPROVED** [review]

**Pull request merged**